dashboard/tests/test_dashboard.py

```diff
@@ -158,10 +158,23 @@ def test_basic(ray_start_with_dashboard):
     assert agent_ports is not None
 
 
+@pytest.mark.skipif(
+    os.environ.get("RAY_MINIMAL") != "1",
+    reason="This specifically tests the minimal installation.",
+)
+def test_missing_imports(shutdown_only):
+    """
+    Test dashboard fails when packages are missing but inclusion
+    was explicitly specified by the user.
+    """
+    with pytest.raises(Exception):
+        ray.init(include_dashboard=True)
+
+
 def test_raylet_and_agent_share_fate(shutdown_only):
     """Test raylet and agent share fate."""
 
-    ray.init(include_dashboard=True)
+    ray.init()
```

Comment:
yep. basically, any check for minimal behavior now needs `include_dashboard=None` instead of `include_dashboard=True` as it will otherwise raise an exception early on